.github/workflows/ci.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       # the tests need the gprofiler image built (from Dockerfile). I run it separately here, because "docker build" prints the build logs
       # more nicely. the tests will then be able to use the built image.
     - name: Build gProfiler image
-      run: DOCKER_BUILDKIT=1 docker build -t gprofiler .
+      run: ./scripts/build_x86_64_container.sh -t gprofiler
```

Comment:
Perhaps also build the ARM one in this workflow?

Reply:
ARM is emulated and takes lots of time (container build takes >3hrs). Not worth the time

Reply:
I think in the future it would be easier to install a cross-compiler for aarch64

Reply:
It will be easier for the processor to run it, but I doubt it'll be easier for us to write it (it's not just cross compiler, it's also getting `pip` to download the correct packages, and getting Rust, and fixing many different Makefiles with their different packages, ...)